Makefile

```diff
@@ -130,7 +130,7 @@ clean: clean-build
 .PHONY: build-setup-deploy
 build-setup-deploy: clean sandbox-issuer-docker sandbox-rp-docker sandbox-ace-rp-docker sandbox-cli-docker login-consent-server-docker
 	@TRUSTBLOC_CORE_K8S_COMMIT=$(TRUSTBLOC_CORE_K8S_COMMIT) \
-		CURRENT_TIME=$(shell date +"%Y%m%dT%H%M%S%z") \
```

Comment:
removed time zone designator because it might generate "+" symbol ( `20210505T193241+0300`) that results in an invalid tag